integration/src/main/java/scripts/testscripts/CloneBigQueryDataset.java

```diff
@@ -94,29 +172,30 @@ protected void doUserJourney(TestUserSpecification testUser, WorkspaceApi worksp
     final String clonedDatasetDescription = "Clone of " + destinationDatasetName;
     final String jobId = UUID.randomUUID().toString();
     final CloneControlledGcpBigQueryDatasetRequest cloneRequest = new CloneControlledGcpBigQueryDatasetRequest()
-        .cloningInstructions(CloningInstructionsEnum.DEFINITION)
+        .cloningInstructions(CloningInstructionsEnum.RESOURCE)
```

Comment:
Not yet. I need to think about the best way to do that so I don't have a lot of redundancy with this test. It would be nice to have it in the CLI for that kind of test.